### DISCUSSION TITLE
.NET 8 and Blazor United / Server-side rendering

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#87] [General]
# .NET 8 and Blazor United / Server-side rendering

New .NET and new Blazor features. In this blog post, I want to highlight the new features that are hitting us with .NET 8 in the Blazor world. So let's see what's new.

https://steven-giesel.com/blogPost/3c71e553-490f-4d52-8104-eb8338c0207f

**Comment by EngincanV** [comment]
Great article as always, thanks Steven. 

> BTW, I have noticed the giesel.engineering website URL is wrong (it's at the beginning of the article).

**Reply by linkdotnet** [reply]
Hey and thanks for the kind words.

I also fixed the URL - the blog interpreted the link as relative instead of an absolut url.
